python/doc/examples/reliability_sensitivity/reliability/plot_FORM_explained.py

```diff
@@ -0,0 +1,450 @@
+# - a standard unit gaussian :math:`X_2 \sim \mathcal{N}(0,1)`.
+#
+# The support of the input vector is :math:`[0, +\infty[ \times \mathbb{R}`
+#
+
+# %%
+distX1 = ot.Exponential(1.0)
+distX2 = ot.Normal()
+distX = ot.ComposedDistribution([distX1, distX2])
+
+
+# %%
+# We can draw the bidimensional PDF of the distribution `distX` over :math:`[0,-10] \times [10,10]` :
+ot.ResourceMap_SetAsUnsignedInteger("Contour-DefaultLevelsNumber", 8)
+graphPDF = distX.drawPDF([0,-10],[10,10])
+graphPDF.setTitle(r'2D-PDF of the input variables $(X_1, X_2)$')
+graphPDF.setXTitle(r'$x_1$')
+graphPDF.setYTitle(r'$x_2$')
+graphPDF.setLegendPosition("bottomright")
+view = otv.View(graphPDF)
+
+
+# %%
+# We consider the model :math:`f : (x_1, x_2) \mapsto x_1 x_2` which maps the random input vector :math:`X` to the output variable :math:`Y=f(X) \in \mathbb{R}`. We also draw the isolines of the model `f`.
+#
+f = ot.SymbolicFunction(['x1', 'x2'], ['x1 * x2'])
+graphModel = f.draw([0.0, -10.0],[10.0,10.0])
+graphModel.setXTitle(r'$x_1$')
+graphModel.setXTitle(r'$x_2$')
+graphModel.setTitle(r'Isolines of the model : $Y = f(X)$')
+view = otv.View(graphModel)
+
+
+# %%
+# We want to estimate the probability :math:`P_f` of the output variable to be greater than a prescribed threshold :math:`s=10` : this is the failure event. This probability is simply expressed as an integral :
+#
+# .. math::
+#
+#    P_f = \int_{\mathcal{D}} \mathbf{1}_{\mathcal{D}}(x) df_{X_1,X_2}(x)
+#
+# where :math:`\mathcal{D} = \{ (x_1, x_2) \in [0,+\infty[ \times \mathbb{R} / x_1 x_2 \geq s \}` is the failure domain.
+# In the general case the probability density function :math:`f_{X_1,X_2}` and the domain of integration :math:`\mathcal{D}` are difficult to handle.
+
+# %%
+# We first define RandomVector objects and the failure event associated to the ouput random variable.
+vectorX = ot.RandomVector(distX)
+vectorY = ot.CompositeRandomVector(f, vectorX)
+s = 10.0
+event = ot.ThresholdEvent(vectorY, ot.Greater(), s)
+
+
+# %%
+# This event can easily be represented with a 1D curve as it is a branch of an hyperbole.
+# If :math:`y =  x_1 x_2 = 10.0`, then the boundary of the domain of failure is the curve :
+#
+# .. math::
+#
+#    h : x_1 \mapsto \frac{10.0}{x_1}
+#
+
+# %%
+# We shall represent this curve using a :class:`~openturns.Contour` object.
+nx, ny = 15, 15
+xx = ot.Box([nx], ot.Interval([0.0], [10.0])).generate()
+yy = ot.Box([ny], ot.Interval([-10.0], [10.0])).generate()
+inputData = ot.Box([nx,ny], ot.Interval([0.0, -10.0], [10.0, 10.0])).generate()
+outputData = f(inputData)
+mycontour = ot.Contour(xx, yy, outputData, ot.Point([10.0]), ot.Description(["10.0"]))
+myGraph = ot.Graph('Representation of the failure domain', r'$X_1$', r'$X_2$', True, '')
+myGraph.add(mycontour)
+
+# %%
+texts = ot.Description(1)
+texts[0] = r' Event : $\mathcal{D} = \{Y \geq 10.0\}$'
+myText = ot.Text(ot.Point([4.0]), ot.Point([4.0]), texts)
```

Comment:
there should be no need to use Description or Point here, just use plain lists

Reply:
@KieranDelamotte : I cannot see the fix after Julien's suggestion.

Reply:
I have just pushed  a commit for this one.